src/tools/race/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
 find_package(Threads)
+find_package(LibRt)
 
 if (APPLE)
 	remove_tool(race "race tool does not compile under MAC OS X")
 else()
+	if (NOT LIBRT_FOUND)
+		remove_tool(race "race tool requires librt support")
```

Comment:
Are there unit tests for the race tool? There is only `race.c` and `CMakeLists.txt` in `src/tools/race/`.